i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/podprobemarker.md

```diff
@@ -0,0 +1,109 @@
+---
+title: PodProbeMarker
+---
+**FEATURE STATE:** Kruise v1.3.0
+
+Kubernetes提供了三种默认的Pod生命周期管理：
+- **Readiness Probe** 用来判断业务容器是否已经准备好响应用户请求，如果检查失败，会将该Pod从Service Endpoints中剔除。
+- **Liveness Probe** 用来判断容器的健康状态，如果检查失败，kubelet将会重启该容器。
+- **Startup Probe** 用来判断容器是否启动完成，如果定义了该Probe，那么Readiness Probe与Liveness Probe将会在它成功之后再执行。
+
+所以Kubernetes中提供的Probe能力都已经限定了特定的语义以及相关的行为。**除此之外，其实还是存在自定义Probe语义以及相关行为的需求**，例如：
+- **GameServer定义 Idle Probe 用来判断该Pod当前是否存在游戏对局**，如果没有，从成本优化的角度，可以将该Pod缩容掉。
+- **K8S Operator定义 main-secondary Probe 来判断当前Pod的角色（main or secondary）**，升级的时候，可以优先升级 secondary，进而达到升级过程只有一次选主的行为，降低升级过程中服务抖动时间。
+
+OpenKruise提供了自定义Probe的能力，并将结果返回到Pod Status中，用户可以根据该结果决定后续的行为。
+
+## Usage
+```yaml
+apiVersion: apps.kruise.io/v1alpha1
+kind: PodProbeMarker
+metadata:
+  name: game-server-probe
+  namespace: ns
+spec:
+  selector:
+    matchLabels:
+      app: game-server
+  probes:
+  - name: Idle
+    containerName: game-server
+    probe:
+      exec: /home/game/idle.sh
+      initialDelaySeconds: 10
+      timeoutSeconds: 3
+      periodSeconds: 10
+      successThreshold: 1
+      failureThreshold: 3
+    markerPolicy:
+    - state: Succeeded
+      labels:
+        gameserver-idle: 'true'
+      annotations:
+        controller.kubernetes.io/pod-deletion-cost: '-10'
+    - state: Failed
+      labels:
+        gameserver-idle: 'false'
+      annotations:
+        controller.kubernetes.io/pod-deletion-cost: '10'
+    podConditionType: game.io/idle
+```
+
+- **spec.selector**: 根据Label选择匹配的Pods，MatchLabels和MatchExpressions都支持。详情请参考：https://kubernetes.io/docs/concepts/overview/working-with-objects/labels
```

Comment:
plz comment that once specified, selector cannot be changed for a PodProbeMarker